app/src/app/api/v0/city/invite/[invite]/route.ts

```diff
@@ -35,15 +36,12 @@ export const GET = apiHandler(async (req, { params, session }) => {
     },
   });
   const host = process.env.HOST ?? "http://localhost:3000";
-  if (!user) {
-    return NextResponse.redirect(host);
-  }
 
   const city = await db.models.City.findOne({
     where: { cityId: invite.cityId },
   });
 
   await user?.addCity(city?.cityId);
 
-  return NextResponse.redirect(host);
+  return NextResponse.redirect(`${host}/en/${inventory}`);
```

Comment:
Don't hard-code `en` here. You can leave it out and the middleware will automatically redirect to the correct language for the user while loading the page (that's how it works for all our other links).

Reply:
ahh missed this one